pkg/rancher-desktop/main/snapshots/snapshots.ts

```diff
@@ -15,9 +15,14 @@ class SnapshotsError {
 
   constructor(response: SpawnResult) {
     console.debug(response.stdout);
-    const value = JSON.parse(response.stdout);
 
-    this.message = value?.error;
+    try {
```

Comment:
Missed the previous PR, but I would prefer that `main/snapshots/snapshots.ts` and `store/snapshots.ts` have different base names. Maybe leave the `store/` name as is, and change this one to `snapshotsShim.ts`, as it seems to be mostly an adapter for rdctl.